build/__phpVersions.sh

```diff
@@ -1,15 +1,15 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
-PHP_BUILD_BASE_TAG='20191025.2'
-PHP_RUNTIME_BASE_TAG='20191025.2'
+PHP_BUILD_BASE_TAG='20191104.7'
+PHP_RUNTIME_BASE_TAG='20191104.7'
```

Comment:
```
-------------------------------------BEFORE-----------------------------------------

$ docker system df -v
Images space usage:

REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE                SHARED SIZE         UNIQUE SIZE         CONTAINERS
oryx/php-7.3        latest              9a9ff9a4034f        2 minutes ago       900.8MB             342.4MB             558.5MB             0
oryx/php-7.2        latest              8321845b6c9b        2 minutes ago       899.9MB             342.4MB             557.6MB             0
oryx/php-7.0        latest              744c743f50c7        3 minutes ago       969.8MB             0B                  969.8MB             0
oryx/php-5.6        latest              a8ddd8f77326        3 minutes ago       952.2MB             0B                  952.2MB             0

$ docker system df
TYPE                TOTAL               ACTIVE              SIZE                RECLAIMABLE
Images              4                   0                   3.38GB              3.38GB (100%)
Containers          0                   0                   0B                  0B
Local Volumes       0                   0                   0B                  0B
Build Cache         0                   0                   0B                  0B


-------------------------------------AFTER-----------------------------------------
$ docker system df -v
Images space usage:

REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE                SHARED SIZE         UNIQUE SIZE         CONTAINERS
oryx/php-7.3        latest              102f97536594        2 minutes ago       1.109GB             857.3MB             251.6MB             0
oryx/php-7.2        latest              e425956a40d3        2 minutes ago       1.108GB             857.3MB             250.5MB             0
oryx/php-7.0        latest              734dd49cbde6        2 minutes ago       1.034GB             822.7MB             211.7MB             0
oryx/php-5.6        latest              d2e194e2cea0        3 minutes ago       960.3MB             0B                  960.3MB             0

$ docker system df
TYPE                TOTAL               ACTIVE              SIZE                RECLAIMABLE
Images              4                   0                   2.531GB             2.531GB (100%)
Containers          0                   0                   0B                  0B
Local Volumes       0                   0                   0B                  0B
Build Cache         0                   0                   0B                  0B

```

Reply:
Notice that 5.6 image does not share the layers...this is expected as we are currently unable to use common layers since building it results in errors (possible because of dependency on stale packages which are no longer available)

Reply:
cc @sanchitmehta